frigate/const.py

```diff
@@ -5,3 +5,8 @@
 YAML_EXT = (".yaml", ".yml")
 PLUS_ENV_VAR = "PLUS_API_KEY"
 PLUS_API_HOST = "https://api.frigate.video"
+
+# Regex Consts
+
+REGEX_CAMERA_NAME = "^[a-zA-Z0-9_-]+$"
+REGEX_CAMERA_USER_PASS = "[a-zA-Z0-9_-]+:[a-zA-Z0-9!$_-]+@"
```

Comment:
I know we can't have special characters in password right now but figured might as well filter them out in case that changes.

Reply:
I'm sure we can fix that too. Might as well look at doing that here. They probably just need to be properly escaped when opening the subprocess.